### PR TITLE
Custom message in the config file

### PR DESCRIPTION
**Pull request by hunkim**
#111 
- Add quick example in the README.md
- Added repoConfig.message based ConfigMessageGenerator.

Without config/no message in config:

image

With config file:
```
{
  "maxReviewers": 5, // Maximum  number of people to ping in the PR message, default is 3
  "numFilesToCheck": 10, // Number of files to check against, default is 5
  "message": "@pullRequester, cool! @reviewers, please check this out for @pullRequester."
}
```

image

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -88,6 +88,13 @@ function defaultMessageGenerator(reviewers, pullRequester) {
   );
 }
 
+function configMessageGenerator(message, reviewers, pullRequester) {
+  // replace all @reviewers. message may have more than one '@reviewers'.
```
Comment (vjeux) [inline server.js]: Remove those two comments, they are obvious from reading the line after

File: server.js [hunk 2]
```diff
@@ -177,16 +184,28 @@ async function work(body) {
...
   }
 
-  github.issues.createComment({
-    user: data.repository.owner.login, // 'fbsamples'
-    repo: data.repository.name, // 'bot-testing'
-    number: data.pull_request.number, // 23
-    body: messageGenerator(
+  var message = null;
+  if (repoConfig.message) {
+    // generate message using repoConfig.message if it is set
```
Comment (vjeux) [inline server.js]: Remove

**Comment by vjeux** [comment]
Great! Can you just remove the comments that don't add any value and i'll merge it in. Thanks :)

**Comment by hunkim** [comment]
@vjeux Thanks! Done.

**Pull request merged**